receiver/mysqlreceiver/client.go

```diff
@@ -58,6 +59,24 @@ type IndexIoWaitsStats struct {
 	index string
 }
 
+type PerfEventsStatementsStats struct {
+	schema                    string
+	digest                    string
+	digestText                string
+	countStar                 int64
```

Comment:
I took a `star` from column name, event if it's not so informative. Should I change it to `total`?